src/StatsdClient/StatsdClient.csproj

```diff
@@ -19,7 +19,11 @@
   <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
     <DefineConstants>OS_WINDOWS</DefineConstants>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetFramework)' != 'net451' AND '$(TargetFramework)' != 'net461'">
+  <ItemGroup Condition="'$(TargetFramework)' != 'net45' AND '$(TargetFramework)' != 'net461'">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0">
```

Comment:
I thought the reference assemblies were needed to build `net45` and `net461` when those SDKs are not available (for example, on Linux). But the condition above includes the reference when _not_ targeting those versions. Am I missing something?

Reply:
Fixed.